databricks_cli/configure/provider.py

```diff
@@ -49,7 +49,14 @@ def _create_section_if_absent(raw_config, profile):
 
 
 def _get_option_if_exists(raw_config, profile, option):
-    return raw_config.get(profile, option) if raw_config.has_option(profile, option) else None
+    if profile == DEFAULT_SECTION:
+        # We must handle the DEFAULT_SECTION differently since it is not in the _sections property
```

Comment:
The comment about `_sections`  is hard to understand. We should say `has_option/get_option` would fallback and lookup the DEFAULT section.